tutor/resources/styles/components/task-step/ends.less

```diff
@@ -81,7 +81,8 @@
       content: attr(data-question-number) ")";
       position: absolute;
       z-index: 1;
-      left: -25px;
+      right: 100%;
```

Comment:
Not bigs, but...just checking, this would be the same thing as `left: 0;` right?